scripts/assets/dummy_delta_lake.ipynb

```diff
@@ -0,0 +1,189 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "14c367a4",
+   "metadata": {},
+   "source": [
+    "# Delta lakes: creating a sample store"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e7ffebad",
+   "metadata": {},
+   "source": [
+    "This notebook is intended to create a simple dummy delta lake store compliant with the schema outlined in the ECR validation directory. It leverages `pyspark` and `delta-spark` working within a virtual environment to create an empty table and insert a few dummy rows. It can also be run completely outside of a virtual environment, if desired, in which case only step 4 below need be followed. To configure an environment in order to run this script, perform the following steps:\n",
```

Comment:
```suggestion
    "This notebook is intended to create a simple dummy delta lake store compliant with the schema outlined in the ECR validation directory. It leverages `pyspark` and `delta-spark` working within a virtual environment to create an empty table and insert a few dummy rows. It can also be run completely outside of a virtual environment, if desired, in which case only steps 3 and beyond need be followed. To configure an environment in order to run this script, perform the following steps:\n",
```